src/react/workflow/replication/Replication.jsx

```diff
@@ -30,9 +30,9 @@ const ReplicationContainer = styled.div`
 const schema = Joi.object({
     streamId: Joi.string().label('Id').allow(''),
     streamVersion: Joi.number().label('Version').optional(),
-    streamName: Joi.string().label('Name').min(4).allow('').messages({
-        'string.min': '"Name" should have a minimum length of {#limit}',
-    }),
+    // streamName: Joi.string().label('Name').min(4).allow('').messages({
```

Comment:
not sure whats up here

Reply:
I decided to keep this one because later we will want to let the user have the ability to add a replication name.